Il2CppTests/TestGenerics.cs

```diff
@@ -82,7 +87,11 @@ public partial class FixedTests
                 (tT, "T", false, false, true, true, 0),
                 (tU, "U", false, false, true, true, 1),
                 (tF, "G`1[Derived`1[V]]", true, false, true, false, -1),
-                (tNested, "Derived`1[V]+Nested[V]", true, true, true, false, -1)
+                (tNested, "Derived`1[V]+Nested[V]", true, true, true, false, -1),
+                (tB, "B", false, false, true, true, 0),
+                (tB.BaseType, "Derived`1[R]", true, false, true, false, -1),
+                (tI, "I", false, false, true, true, 1),
+                (tI.ImplementedInterfaces.ElementAt(1), "IEnumerable`1[R]", true, false, true, false, -1),
```

Comment:
I assume you checked all these new test cases against the real output from .NET Reflection APIs?

Reply:
Yeah, everything here (plus the new stuff) matches the real API, *except* that `tNested`'s name is spelled ``Derived`1+Nested[V]`` in the real API. I also checked the methods as well to be complete - those also match completely.